i18n-helpers/src/lib.rs

```diff
@@ -24,10 +24,12 @@
 //! how to use the supplied `mdbook` plugins.
 
 use polib::catalog::Catalog;
-use pulldown_cmark::{Event, LinkType, Tag};
+use pulldown_cmark::{CodeBlockKind, CowStr, Event, LinkType, Tag};
 use pulldown_cmark_to_cmark::{cmark_resume_with_options, Options, State};
 use regex::Regex;
 use std::sync::OnceLock;
+use syntect::easy::ScopeRangeIterator;
+use syntect::parsing::{BasicScopeStackOp, ParseState, Scope, ScopeStack, SyntaxSet};
```

Comment:
By the way, I would be happy to move this new code block parsing logic to its own little `code_blocks.rs` module (before/after) merging this PR.

Reply:
Let's do this after merging the PR.